### PR TITLE
simpler Gemfile and config files

### PR DESCRIPTION
**Pull request by langdon**
experimenting with a simpler Gemfile and config files hoping it will be more tolerant to upgrades. Also adds to the README for upgrading and running dev mode.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+3. `git clone` the repo somewhere.
+4. Open the folder in VS Code.
+5. In a terminal run `make build` or `make build-podman` as appropriate.
+6. run `make run` or `make run-podman`.
+7. You should be able to go to a browser and goto [http://localhost:4000](http://localhost:4000) and see the site.
+
+### Notes
+* You should always be making a branch for your changes
+* You should not use a fork as then the preview automation won't work
+* Once you are pretty happy with your code, create a PR from your branch (after you pushed to GitHub) and add `WIP` to the beggining of the name of the PR. Go to your PR and you should see a link to preview of the site (it can take awhile).
```
Comment (oeaugustin) [inline README.md]: beggining => beginning

File: README.md [hunk 2]
```diff
@@ -1,2 +1,45 @@
...
+
+### Via GitHub Codespaces
+By far the easiest mechanism to make changes.
+
+1. Click on the big green "<> Code" button, click to the 2nd tab (Codespaces), click the "+" sign.
+2. Once it is done warming up, you should be able to open a terminal and run `./launch.sh`.
+3. However, if you get an error about dependencies, run `bundle update` then run `./launch.sh`.
+4. Copy the url to your codespace which will look something like `https://obscure-dollop-qsdadASD.github.dev/`
+5. Open a new browser window or tab and paste the url.
+6. Before hitting enter modify it to be `https://obscure-dollop-qsdadASD-4000.app.github.dev/` (NOTE: you are adding `-4000.app` kind of in the middle).
```
Comment (oeaugustin) [inline README.md:26]: ...hitting enter, modify... (add comma after enter for easier reading)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,45 @@
...
+We are now using [`pages-gem`](https://github.com/github/pages-gem) and [`just-the-docs`](https://just-the-docs.com/) as our only gem dependencies and, hopefully, it will upgrade more nicely.
+
+## Running the application
+
+### Production
+The github pages site will update automatically on every commit to main.
+The jekyll config is managed in `/_config.prod.yml` as you can see in the [GitHub Workflow](https://github.com/BU-Spark/bu-spark/blob/main/.github/workflows/build-jekyll.yml).
+The use of a non-standard file is to support a need for a separate config (in the standard place) for dev.
+
+## Development
```
Comment (oeaugustin) [inline README.md:16]: should this be ###Development? on the same level with Production

File: README.md [hunk 4]
```diff
@@ -1,2 +1,45 @@
...
+
+### Via GitHub Codespaces
```
Comment (oeaugustin) [inline README.md:18]: And this one more level down, #### Via Github Codespaces

File: README.md [hunk 5]
```diff
@@ -1,2 +1,45 @@
...
+1. Install `docker` or `podman` and `make`.
+2. Find the URI for the repo under the big green "<> Code" button.
+3. `git clone` the repo somewhere.
+4. Open the folder in VS Code.
+5. In a terminal run `make build` or `make build-podman` as appropriate.
+6. run `make run` or `make run-podman`.
+7. You should be able to go to a browser and goto [http://localhost:4000](http://localhost:4000) and see the site.
+
+### Notes
+* You should always be making a branch for your changes
```
Comment (oeaugustin) [inline README.md:43]: make a new branch

File: README.md [hunk 6]
```diff
@@ -1,2 +1,45 @@
...
+* You should not use a fork as then the preview automation won't work
```
Comment (oeaugustin) [inline README.md:44]: fork(,)  add comma

**Comment by oeaugustin** [comment]
Preview works and readme looks good. Just added some suggestions, resolve if you deem unnecessary

**Comment by langdon** [comment]
all resolved (in theory)

**Pull request merged**